.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 10.x
```

Comment:
10 is pretty old and not maintained anymore, maybe we could switch to 12 or 14? 
https://nodejs.org/en/about/releases/

Reply:
the TravisCI config was using 10 and I tried to port it as close as possible